<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCenterWindow</class>
 <widget class="QWidget" name="CreateCenterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/add_icon.png</normaloff>:/icons/add_icon.png</iconset>
  </property>
  <widget class="QPushButton" name="applyPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>235</y>
     <width>391</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Створити центр</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="loginGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Введіть дані про центр</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Адреса</string>
    </property>
    <widget class="QLabel" name="houseLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер будинку</string>
     </property>
    </widget>
    <widget class="QLabel" name="streetLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Вулиця</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="streetLineEdit">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>30</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="houseLineEdit">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>70</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Кількість місць</string>
    </property>
    <widget class="QLineEdit" name="generalPlacesLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="generalPlacesLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Загальна кіл-сть місць</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
